lib/Thumbor/Url.php

```diff
@@ -44,6 +44,11 @@ public function stringify($server, $secret, $original, $commands)
             $imgPath = $original;
         }
 
+        if(is_array($server)) {
+            $index = strlen($imgPath) % count($server);
```

Comment:
I personally don't think that this belongs within this library.

A fairly small wrapper class containing an array of servers and a method to map `$original` to server could probably suffice. Maybe something along the lines of:

```php
<?php

use Thumbor\Url;

class PhumborServerFarm
{
    /** @var array */
    private $servers;

    /**
     * @param array $servers
     * @param string $secret
     */
    public function __construct($servers, $secret)
    {
        $this->servers = $servers;
    }

    /**
     * @param string $original
     * @param array $commands
     */
    public function stringify( $original, $commands)
    {
        return Url::stringify($this->getServer($original), $secret, $original, $commands);
    }

    private function getServer($original)
    {
        // return server here
    }
}
```